agents/kubevirt/fence_kubevirt.py

```diff
@@ -0,0 +1,123 @@
+#!/usr/bin/python -tt
```

Comment:
This should be `#!@PYTHON@ -tt` so it gets set based on detected or specified path of the binary.

Reply:
I pushed it by mistake after changed for testing, thanks